common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -6,9 +6,8 @@
 
 import json
 import logging
-import warnings
-
 from lxml import etree
+import warnings
```

Comment:
Minor: the `warnings` module is part of the standard library, so it should be left where it was.